monai/data/dataset.py

```diff
@@ -173,6 +176,17 @@ def __init__(
             if not self.cache_dir.is_dir():
                 raise ValueError("cache_dir must be a directory.")
 
+    def update_data(self, data: Sequence):
```

Comment:
LMDBDataset inherits this but probably wouldn't work

Reply:
Cool, thanks for your finding!
I added support to LMDBDataset in the latest commit and unit tests.

Thanks.